src/lib.rs

```diff
@@ -84,3 +106,44 @@ fn makecircle(centerpoint: Point<f64>, radius: f64, num_vertices: usize) -> Poly
     let polygon = Polygon::new(LineString::from(circle_points), vec![]);
     polygon
 }
+
+// Make a single clock polygon
+pub fn clockpoly(
```

Comment:
Default to keeping things as private as possible. It's unlikely somebody using the API wants just one of these, right?